main8.py

```diff
@@ -0,0 +1,12 @@
+"""Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+*Пример:*
+3 2 4 -> yes
+3 2 1 -> no
+"""
+n = 3
+m = 8
+k = 24
+if(k<= m * n and k % m ==0) or (k<= m * n and k % n== 0):
```

Comment:
Как вы можете заметить, одно и то же условие повторяется слева и справа от `or`. Значит, его можно как бы вынести за скобки.
```suggestion
if 0 < k < m * n and (k % m == 0 or k % n == 0):
```